package/MDAnalysis/core/universe.py

```diff
@@ -460,7 +488,7 @@ def empty(cls, n_atoms, n_residues=1, n_segments=1,
         top = Topology(n_atoms, n_residues, n_segments,
                        atom_resindex=atom_resindex,
                        residue_segindex=residue_segindex,
-        )
+                       )
```

Comment:
```suggestion
        )
```

It is a matter of taste, but I do prefer the parenthesis where it was. By being at the same level of indentation as the line that opens the parenthesis, the closing parenthesis looks like closing a block which I, personally, find clearer.